packages/x-tree-view/src/internals/plugins/useTreeViewExpansion/useTreeViewExpansion.types.ts

```diff
@@ -2,10 +2,20 @@ import * as React from 'react';
 import { DefaultizedProps, TreeViewPluginSignature } from '../../models';
 import { UseTreeViewItemsSignature } from '../useTreeViewItems';
 
-export interface UseTreeViewExpansionInstance {
+export interface UseTreeViewExpansionPublicAPI {
+  /**
+   * Change the expansion status of a given item.
+   * @param {React.SyntheticEvent} event The UI event that triggered the change.
+   * @param {string} itemId The id of the item to modify.
+   * @param {boolean} isExpanded The new expansion status of the given item.
+   */
+  setItemExpansion: (event: React.SyntheticEvent, itemId: string, isExpanded: boolean) => void;
+}
+
+export interface UseTreeViewExpansionInstance extends UseTreeViewExpansionPublicAPI {
   isItemExpanded: (itemId: string) => boolean;
   isItemExpandable: (itemId: string) => boolean;
-  toggleItemExpansion: (event: React.SyntheticEvent, value: string) => void;
+  toggleItemExpansion: (event: React.SyntheticEvent, itemId: string) => void;
```

Comment:
Those methods are private and don't have JSDoc for now.
I think it would be nice to document them in the future as the team grows, but for now I'm focusing on having a good JSDoc for all the public methods.